src/app/home/home.component.scss

```
@@ -7,6 +7,27 @@ span.access-code{
   cursor: pointer;
 }
 
+.nav-group{
+  overflow-y: auto;
+  overflow-x: hidden;
+}
+
+.nav-event-name-short{
+  width: 80%;
+  text-overflow: ellipsis;
+  overflow: hidden;
+  white-space: nowrap;
+  padding: 0 0 0 10px;
+}
+
+.nav-event-name-long{
+  width: 100%;
+  text-overflow: ellipsis;
+  overflow: hidden;
+  white-space: nowrap;
+  padding: 0 0 0 10px;
+}
```

The duplication in above code can be easily avoided by using a base and a marker class. For example:

```suggestion
.nav-event-name {
  width: 100%;
  text-overflow: ellipsis;
  overflow: hidden;
  white-space: nowrap;
  padding: 0 0 0 10px;
}

.nav-event-name.short {
  width: 80%;
}
```